packages/react-is/src/__tests__/ReactIs-test.js

```diff
@@ -152,6 +156,14 @@ describe('ReactIs', () => {
     expect(ReactIs.isMemo(Component)).toBe(false);
   });
 
+  it('should identify lazy', () => {
+    const Component = () => React.createElement('div');
+    const lazyComponent = React.lazy(() => Component);
```

Comment:
nit: This should actually be `React.lazy(() => ({default: Component});` but that doesn't matter for the purposes of this test.